data/src/main/java/com/globant/equattrocchio/data/ImageReporsitory.java

```diff
@@ -0,0 +1,36 @@
+package com.globant.equattrocchio.data;
+
+import com.globant.equattrocchio.data.entities.ImageEntity;
+import com.globant.equattrocchio.domain.enities.Image;
+
+import java.util.List;
+
+import io.realm.Realm;
+
+
+public class ImageReporsitory {
```

Comment:
ImageReporsitory typo and misleading name should be: ImagesRepository

Reply:
done